src/styles/brackets.less

```diff
@@ -1110,16 +1091,15 @@ a, img {
         content: " ";
         left: 0;
         z-index: @z-index-brackets-inline-editor-shadow;
-    }
-
-    .shadow.top {
-        top: 0px;
-        background-image: -webkit-linear-gradient(top, rgba(0, 0, 0, 0.1), rgba(0, 0, 0, 0));
-    }
-
-    .shadow.bottom {
-        bottom: 0px;
-        background-image: -webkit-linear-gradient(top, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.1));
+        
+        &.top {
+            top: 0;
+            background-image: -webkit-linear-gradient(top, rgba(0, 0, 0, 0.1), rgba(0, 0, 0, 0));
```

Comment:
We should be able to use the unprefixed version since it works since Chrome 26, and using the new syntax. (`to bottom` instead of `top`)